<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VMTDNxApiAdapterForm</class>
 <widget class="QWidget" name="VMTDNxApiAdapterForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>967</width>
    <height>662</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <layout class="QHBoxLayout" name="_4">
     <item>
      <widget class="QPlainTextEdit" name="pteFlow">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>1</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="sizeIncrement">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbHideRight">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>20</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>20</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>&lt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="wRight" native="true">
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QGroupBox" name="gbConnection">
          <property name="title">
           <string>Подключение</string>
          </property>
          <layout class="QGridLayout" name="gridLayout">
           <item row="4" column="0">
            <widget class="QLabel" name="_10">
             <property name="text">
              <string>Пароль:</string>
             </property>
            </widget>
           </item>
           <item row="4" column="1">
            <widget class="QLineEdit" name="lePassword"/>
           </item>
           <item row="5" column="1">
            <widget class="QSpinBox" name="sbTicketTimeoutInterval">
             <property name="maximum">
              <number>999999999</number>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="_9">
             <property name="text">
              <string>Пользователь:</string>
             </property>
            </widget>
           </item>
           <item row="5" column="0">
            <widget class="QLabel" name="_14">
             <property name="text">
              <string>Ожид. квитанции (мс):</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0" colspan="2">
            <widget class="QPushButton" name="pbCheckConnection">
             <property name="text">
              <string>Проверка подключения</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QLineEdit" name="leUserName"/>
           </item>
           <item row="2" column="1">
            <widget class="QLineEdit" name="leUrl"/>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="_8">
             <property name="text">
              <string>Ссылка:</string>
             </property>
            </widget>
           </item>
           <item row="6" column="0" colspan="2">
            <layout class="QHBoxLayout" name="_13">
             <property name="topMargin">
              <number>0</number>
             </property>
             <item>
              <spacer name="_12">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QPushButton" name="pbChange">
               <property name="text">
                <string>Изменить</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pbAccept">
               <property name="text">
                <string>Принять</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pbCancel">
               <property name="text">
                <string>Отмена</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="_11">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="_15">
             <property name="text">
              <string>Подключение:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLabel" name="lbIsConnected">
             <property name="text">
              <string>TextLabel</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="gbCommands">
          <property name="title">
           <string>Генератор запросов</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QPlainTextEdit" name="pteCommands"/>
           </item>
           <item>
            <layout class="QHBoxLayout" name="_1">
             <item>
              <spacer name="_3">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QPushButton" name="pbSendCommands">
               <property name="text">
                <string>Отправить</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="_2">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <spacer name="_5">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
